module/programUsers/helper.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * name : helper.js
+ * author : prajwal
+ * created-date : 10-Jun-2024
+ * Description : Programs users related helper functionality.
+ */
+
+// Dependencies
+
+/**
+ * ProgramUsersHelper
+ * @class
+ */
+
+const programUsersQueries = require(DB_QUERY_BASE_PATH + '/programUsers')
+
+module.exports = class ProgramUsersHelper {
+	/**
+	 * check if user joined a program or not and consentShared
+	 * @method
+	 * @name checkForUserJoinedProgramAndConsentShared
+	 * @param {String} programId
```

Comment:
@Prajwal17Tunerlabs give explanation to each param

Reply:
params explanation added